sink/src/it/java/com/datastax/oss/kafka/sink/simulacron/SimpleEndToEndSimulacronIT.java

```diff
@@ -87,26 +88,29 @@
 class SimpleEndToEndSimulacronIT {
 
   private static final String INSERT_STATEMENT =
-      "INSERT INTO ks1.table1(a,b) VALUES (:a,:b) USING TIMESTAMP :kafka_internal_timestamp";
-  // if user provided custom query, it does not have auto-generated :kafka_internal_timestamp
+      "INSERT INTO ks1.table1(a,b) VALUES (:a,:b) USING TIMESTAMP :" + SinkUtil.TIMESTAMP_VARNAME;
```

Comment:
why `kafka_internal_timestamp` is extracted to `SinkUtil` whereas the `message_internal_timestamp` is not?

Reply:
@tomekl007 
I left the current value only in the comment below. for all of the rest of the code I am using the two new constants  `SinkUtil.TIMESTAMP_VARNAME` and `SinkUtil.TTL_VARNAME`